test/integration/deploy_trigger_test.go

```diff
@@ -197,8 +206,19 @@ func TestTriggers_imageChange(t *testing.T) {
 		t.Fatalf("Expected deployment annotated with deploymentConfig '%s', got '%s'", e, a)
 	}
 
+	// expecting RC update from new to pending
```

Comment:
@deads2k updated, please see the 2nd commit and let me know if that looks better